update.sh

```diff
@@ -21,7 +21,8 @@ declare -A extras=(
   [apache]='RUN set -ex; \
     a2enmod rewrite remoteip; \
     { \
-     echo RemoteIPHeader X-Real-IP; \
+     echo RemoteIPHeader X-Forwarded-For; \
+     echo RemoteIPTrustedProxy 127.0.0.0/8; \
      echo RemoteIPTrustedProxy 10.0.0.0/8; \
      echo RemoteIPTrustedProxy 172.16.0.0/12; \
```

Comment:
Ah okay... we're trusting [RFC1819](https://datatracker.ietf.org/doc/html/rfc1918) private networks already, that answers my interrogation back on the [issue thread](https://github.com/friendica/docker/pull/292#issuecomment-2654662311)

Reply:
That is the apache image only. And I need to figure out how ipv6 is supported.